mne/annotations.py

```diff
@@ -176,6 +176,48 @@ def save(self, fname):
         with start_file(fname) as fid:
             _write_annotations(fid, self)
 
+    def delete_out_of_absolute_time_bounds(self, tmin=None, tmax=None):
```

Comment:
can we call it crop like for other containers?

Reply:
yes !!! that was the name I was missing!